dashboard/app/views/layouts/_student_information_interstitial.html.haml

```diff
@@ -6,7 +6,7 @@
       %p{style: 'font-size: 13px; line-height: 18px; color: #333'}
         = t('parent_mailer.parent_email_added_subject')
         = t('activerecord.attributes.user.finish_sign_up_header_account_information')
-        = t('age_interstitial.disclaimer')
+        = t('age_interstitial.disclaimer', url: CDO.code_org_url('/privacy'), markdown: :inline).html_safe
```

Comment:
This privacy URL is available [in shared_constants.rb](https://github.com/code-dot-org/code-dot-org/blob/staging/lib/cdo/shared_constants.rb#L635). If we use this constant, then if the url ever changes we only have to change it in one place. Seems like overkill but for CPA we had to iterate these URLs many times.

Reply:
Ill make this change thanks!